src/apps/frontend/components/form-control/index.tsx

```diff
@@ -3,7 +3,7 @@ import React, { PropsWithChildren } from 'react';
 import VerticalStackLayout from '../layouts/vertical-stack-layout';
 
 interface FormControlProps {
-  error: string;
+  error: string | false | undefined;
```

Comment:
Why would this be false?

This should be `error?: string` or `error: Nullable<string>`